_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+For this tutorial we will select the most common `components` category.
+
+### Supported Brands
+
+Component brands facilitate [component customisation](/spec/v1/sass/#customisation). Brands change the appearance of component elements globally, e.g. change the appearance of all “primary” buttons, including where they are used by other components. Brands include `master` (think, ft.com pink), `internal` for internal tools and products, and `whitelabel` for a striped-back un-opinionated style. Origami components may support one or more brands. We'll discuss brands more later, for now select the `master`, `internal`, and `whitelabel` brand when prompted by `obt init`.
+
+### Support Status
+
+All Origami components have a support status. For example a `maintained` component means that "[the component is] not actively developed but reproducible bugs will be fixed promptly and work done where necessary to maintain compatibility with platforms and other projects"; an `experimental` component means "the [component] is not ready for production use". For a full list a support statuses and their meaning see the [support status section of the specification](/spec/v1/manifest/#supportstatus). It is sometimes a good idea to release a new component as `experimental` for a trial period to gather feedback from users, so select `experimental` for now.
+
+As well as a support status, the `obt init` command will ask for a support email address and slack channel. This will help identify your team as owners of the component. When the component is published a message will display in the component registry letting potential users know that support is not guaranteed by the Origami team, and that they should contact your team before using the component.
+
+### JavaScript And Sass
+
+Origami components may include JavaScript for interactivity or [Sass](https://sass-lang.com/) for visual styles (we compile [Sass](https://sass-lang.com/) to CSS). When prompted, select that you would like the component to include JavaScript and Sass for the purposes of this tutorial. We will use the code it generates as a base to discuss building an Origami component later.
+
+### Other Questions
+
+This tutorial won't expand on other questions asked by the `obt init` command, like what the component description and keywords are. Please feel free to contact the Origami team if you have any questions during this step.
+
+### Output
+
+Now run `obt init` and answer the questions as above. You should see a directory which contains component boilerplate (commonly shared component code we can build on top of).
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-init-output.png" />
```

Comment:
this image is missing the GitHub team section. Should we add it as a paragraph above as well?

Reply:
I think it's ok to skip this, as I don't think what a GitHub team is needs more explanation outside the cli prompt. The previous section "Other Questions" doesn't list all the questions but is intended as a heads up for other questions not mentioned